packages/mongodb-rag-ingest/src/sources/mongodb-university/makeUniversityPages.ts

```diff
@@ -45,6 +45,27 @@ function makeCatalogItemPages({
 }): Page[] {
   const pages: Page[] = [];
   for (const catalogItem of tiCatalogItems) {
+    /** Create page for higher level courses.
```

Comment:
```suggestion
    /* Create page for higher level courses.
```

only `/**` for doc comments. see https://tsdoc.org/